README.md

```diff
@@ -62,7 +73,9 @@ module.exports = {
     ]}
   },
   responsiveLoader: {
-    sizes: [300, 600, 1200, 2000]
+    sizes: [300, 600, 1200, 2000],
```

Comment:
I haven't really worked with v2 much and will leave updating documentation to someone more knowledgable. We would need to show examples for both versions, as webpack 2 isn't officially out yet.